src/strings/PowerFxResources.en-US.resx

```diff
@@ -4377,4 +4377,26 @@
     <value>**Disclaimer:** AI-generated content can have mistakes. Make sure it's accurate and appropriate before using it. [See terms](https://go.microsoft.com/fwlink/?linkid=2225491)</value>
     <comment>The disclaimer we show on AI functions. This is github flavored markdown. So ** marks text in bold. </comment>
   </data>
+  <data name="AboutUnichar" xml:space="preserve">
+    <value>Converts number to Unicode string</value>
```

Comment:
Suggestion: "Returns the Unicode character that is referenced by the given numeric value."